e2e/various-suite/exemplars.spec.ts

```diff
@@ -11,7 +11,7 @@ const addDataSource = () => {
       e2e.components.DataSource.Prometheus.configPage.exemplarsAddButton().click();
       e2e.components.DataSource.Prometheus.configPage.internalLinkSwitch().check({ force: true });
       e2e.components.DataSource.Prometheus.configPage.connectionSettings().type('http://prom-url:9090');
-      e2e.components.DataSourcePicker.inputV2().click().should('have.focus');
+      cy.get('[data-testid="data-testid Data source picker select container"]').click();
```

Comment:
Is this `data-testid` correct? It looks like there is something wrong to me.

Reply:
It is correct :), all of our `data-testid` attributes need the "data-testid" prefix to work as selectors in our e2e tests.